Blackjack/Controller.cs

```diff
@@ -161,12 +139,44 @@ private void DealHand()
             return card;
         }
 
-        private bool ShouldTurnEnd(Choice choice, int score) 
+        private void DisplayPlayerInformation(int playerScore)
+        {
+            if (Rules.IsBlackjack(playerScore))
+            {
+                _output.WriteLine(String.Format(Messages.Player, Messages.Blackjack, OutputFormatter.DisplayHand(_player.Hand)));
+                return;
+            } 
+            if (Rules.IsBust(playerScore))
+            {
+                _output.WriteLine(String.Format(Messages.Player, Messages.Bust, OutputFormatter.DisplayHand(_player.Hand)));
+                return;
+            }
+            else
+                _output.WriteLine(String.Format(Messages.Player, playerScore, OutputFormatter.DisplayHand(_player.Hand)));
+        }
+
+        private bool ShouldTurnEnd(Choice choice, int score)
         {
-            if (choice == Choice.Stay ) return true; 
+            if (choice == Choice.Stay) return true;
             if (Rules.IsBlackjack(score)) return true;
             if (Rules.IsBust(score)) return true;
             return false;
         }
+
+        private void DisplayDealerInformation(int dealerScore)
```

Comment:
This logic is similar to the player one - the same bust and and blackjack rules apply. Perhaps an opportunity to consolidate `DisplayPlayerInformation` and `DisplayDealerInformation`?